python/lsst/ts/atspectrograph/atspec_csc.py

```diff
@@ -138,16 +144,16 @@ async def end_enable(self, data):
             raise e
 
         try:
-            # Check/Report Grating Wheel position.
+            # Check/Report Grating/Disperser Wheel position.
             state = await self.model.query_gw_status(self.want_connection)
             self.log.debug(f"query_gw_status: {state}")
-            grating_name = str(list(self.model.gratings.keys())[int(state[1])])
+            grating_name = str(list(self.model.gratings_to_enum_mapping_dict.keys())[int(state[1])])
```

Comment:
Same as for `filter_to_enum_mapping_dict`. I would recommend just `gratings_to_enum`.